org.mwc.cmap.NarrativeViewer/src/org/mwc/cmap/NarrativeViewer/NarrativeViewerModel.java

```diff
@@ -483,7 +528,16 @@ public int getColumnWidth()
 
     public Object getProperty(final NarrativeEntry entry)
     {
-      return myTimeFormatter.format(entry.getDTG());
+      HiResDate dtg = entry.getDTG();

```

Comment:
@IanMayo  with weakhashmap it use HiResDate object hashcode so if same value on two records it will use same formating as HiResDate override hashcode